spec/rdiff-backup-test/server_spec.rb

```diff
@@ -0,0 +1,276 @@
+require_relative '../spec_helper'
+
+describe 'rdiff-backup-test::server' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      let(:runner) do
+        ChefSpec::SoloRunner.new(
+          pltfrm.dup.merge(step_into: ['rdiff-backup'])#, ['nrpe-check'])
+        )
+      end
+      cached(:chef_run) { runner.converge(described_recipe) }
+
+      before do
+        stub_command('secrets').and_return(false)
+        Chef::EncryptedDataBagItem.stub(:load).with('rdiff-backup-secrets',
+                                                    'secrets').and_return(
+                                                      key: 'secret-key'
+                                                    )
+      end
+
+      #Delete
+      # it do
+      #   expect(chef_run).to remove_nrpe_check('check_rdiff_job_test1')
+      # end
+      # context 'Delete: nrpe is false' do
+      #   let(:runner) do
+      #     ChefSpec::SoloRunner.new(
+      #       pltfrm.dup.merge(step_into: ['nrpe-check'])
+      #     )
+      #   end
+      #   before do
+      #     chef_run.node.default['rdiff-backup']['nrpe'] = false
```

Comment:
@ramereth Im still having difficulty doing the nrpe tests. I looked in the upstream cookbook to see if they have any matchers but they did not write any tests for the nrpe_check, all I could find on the checks was these. `https://github.com/schubergphilis/nrpe/blob/master/test/fixtures/cookbooks/chef_nrpe_test/recipes/default.rb`  `https://github.com/schubergphilis/nrpe/blob/master/resources/check.rb`  Could I make a matcher for this?